### PR TITLE
Add remove to do list

### PR DESCRIPTION
**Pull request by saied2035**
### Project requirements
✅  Removed all hardcoded items from the tasks array.
✅ Created a new JavaScript file for the new functionality.
✅ Implemented a function for adding a new task (add a new element to the array).
✅ Implemented a function for deleting a task (remove an element from the array).
✅ Implemented a function for editing task descriptions.
✅ By default new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new - array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
 this is a demonstration for indexes point:
[indexes explanation video](https://www.loom.com/share/13bac3953b644f638b4f7d8771c3a484).

✅ Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
✅ All changes to the To-Do List should be saved in local storage.

Thanks for reviewing my PR. 👍

### CODE REVIEW

File: src/modules/events.js [hunk 1]
```diff
@@ -1,5 +1,60 @@
-const taskCompleted = () => {
+/* eslint-disable linebreak-style */
+import { getTasks } from './functions.js';
 
+export const taskAdd = (event, list) => {
```
Comment (elyor-doniyorov) [inline src/modules/events.js:4]: - [x] Kindly make sure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). Take a look at the screenshot below, you will see there is a 5th task is added but its index is set to 4. Kindly make changes,
Screenshot (200)
 please.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @saied2035, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/events.js (hunk 1 above)
Comment (saied2035) [reply src/modules/events.js:4]: Hello Elyor.
Thanks for reviewing my code.
Is it possible to meet on zoom call now?

Comment (saied2035) [reply src/modules/events.js:4]: this is a demonstration for indexes point:
[indexes explanation video](https://www.loom.com/share/13bac3953b644f638b4f7d8771c3a484)

Comment (Meltrust) [reply src/modules/events.js:4]: Hey @saied2035, I'm your new reviewer. Saw your video. I contacted you on Slack.

Comment (saied2035) [reply src/modules/events.js:4]: @Meltrust 
I responded on slack.

**Review by Meltrust — APPROVED** [review]
_Note: I cloned the project.  Upon review, I noticed that the project works as intended.   Upon deleting a task, or a group of tasks, the "index" properties get updated accordingly.   The student left a [video](https://www.loom.com/share/13bac3953b644f638b4f7d8771c3a484) response on the last review._ 

Hi @saied2035,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**